lib/routes/account.js

```diff
@@ -37,6 +38,32 @@ module.exports = function (
 
   var verificationReminder = require('../verification-reminders')(log, db)
 
+  // TODO: this is a wrapper around geodb, not sure where to place this,
+  // so for now this stays here
+  function getGeoData (ip, request) {
+    return geodb(ip)
+            .then(function (loc) {
+              var logEventPrefix = 'fxa.location.accuracy.'
+              var logEvent = ''
+              var accuracy = loc.accuracy
+              switch (true) {
+                case accuracy > 200:
+                  logEvent = 'unknown'
+                  break
+                case accuracy > 25 && accuracy <= 200:
+                  logEvent = 'uncertain'
+                  break
+                case accuracy <= 25:
+                  logEvent = 'confident'
+                  break
+              }
```

Comment:
the switch above looks a bit wacky. do we need a default case? can we just use `if` and avoid `switch`?